test/e2e_node/environment/setup_host.sh

```diff
@@ -64,4 +64,13 @@ fi
 # delete init kubelet from containervm so that is doesn't startup
 if [ -f /etc/init.d/kubelet ]; then
   sudo rm /etc/init.d/kubelet
-fi
\ No newline at end of file
+fi
+
+# Setup systemd if required
+sudo mkdir -p /etc/systemd/system.conf.d
```

Comment:
is there a reason you dropped the if check here: 
https://github.com/kubernetes/kubernetes/pull/26289/files#diff-6cfc24c58556b86ef8ff98e630f74f35R33